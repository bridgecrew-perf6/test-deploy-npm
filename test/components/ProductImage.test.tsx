import React from 'react';
import renderer from 'react-test-renderer'
import ProductCard from '../../src/components/ProductCard'
import ProductImage from '../../src/components/ProductImage'
import { product2 } from './data/products';

describe('ProductImage', () => {
  test('should match snapshot with custom img', () => {
    const wrapper = renderer.create(<ProductImage img='./coffee-img.png' />)
    expect(wrapper.toJSON()).toMatchSnapshot()
  })
  
  test('should match snapshot', () => {
    const wrapper = renderer.create(
      <ProductCard product={product2} >
        {() => (
          <ProductImage/>
        )}
      </ProductCard>
    )
    expect(wrapper.toJSON()).toMatchSnapshot()
  })
  
})
