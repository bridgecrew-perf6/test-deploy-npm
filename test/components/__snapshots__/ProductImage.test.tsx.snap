// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductImage should match snapshot 1`] = `
<div
  className="productCard "
>
  <img
    alt="Product"
    className="productImg "
    src="./coffee-mug.png"
  />
</div>
`;

exports[`ProductImage should match snapshot with custom img 1`] = `
<img
  alt="Product"
  className="productImg "
  src="./coffee-img.png"
/>
`;
